src/Views/batchprocessor.html

```diff
@@ -15,7 +15,32 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
     <div style="padding-top: 10px;">
 
         <div id="submitBatch" ng-if="vm.selectedBatchProcessorTabName==='submitBatch'">
-            <p>This is the submit batch processor tab</p>
+            <div id="selectRegionID">
+                <label>Select Region / State</label>
+                <select ng-model="selectedRegion" ng-options="x.name for x in vm.regionList | orderBy: 'name'"></select>
+            </div>
+            <div>
+                <span>
+                    <label class="chx">
+                        <input id="cbDelineate", type="checkbox" ng-checked="selectedRegion">
+                        <span>&nbsp;&nbsp;Basin Delineation&nbsp;&nbsp;&nbsp;</span>
+                    </label>
+                </span>
+                <span>
+                    <label class="chx">
+                        <input type="checkbox">
+                        <span>&nbsp;&nbsp;Compute Basin Characteristics&nbsp;&nbsp;&nbsp;</span>
+                    </label>
+                </span>
+                
+                <span>
+                    <label class="chx">
+                        <input type="checkbox">
+                        <span>&nbsp;&nbsp;Compute Flow Statistics</span>
+                    </label>

```

Comment:
When I check a checkbox, the checkbox moves upwards slightly. Is there a way to prevent this? 

Kind of hard to see in the image, but the first checkbox is slightly higher than the others:
![image](https://user-images.githubusercontent.com/40237491/217986504-243b1307-94e9-42b2-bbba-cac9ca766d8c.png)